requirements/docs.txt

```diff
@@ -5,9 +5,13 @@ pandoc>=1.0, <=2.2
 docutils>=0.16, <0.20
 sphinxcontrib-fulltoc>=1.0, <=1.2.0
 sphinxcontrib-mockautodoc
+sphinx-autobuild
 sphinx-autodoc-typehints>=1.16
 sphinx-paramlinks>=0.5.1, <=0.5.4
 sphinx-togglebutton>=0.2, <=0.3.2
 sphinx-copybutton>=0.3, <=0.5.0
 sphinx-multiproject
+sphinx-toolbox==3.2.0
+sphinx-rtd-dark-mode
 jinja2>=3.0.0,<3.1.0
+pt-lightning-sphinx-theme @ https://github.com/Lightning-AI/lai_sphinx_theme/archive/master.zip
```

Comment:
I would suggest to add this here or in `requirements/*/docs.txt`, but not both. I don't know which place is better. Maybe @Borda knows.

Reply:
it is already there